src/main/java/com/github/dadrus/App.java

```diff
@@ -7,10 +7,14 @@
 public class App 
 {
 	// expecting sonar to alert an issue here
-	public static final String PASSWORD = "boo";
+	public static final String OTHER_PASSWORD = "boo";
```

Comment:
![BLOCKER](https://sonarsource.github.io/sonar-github/severity-blocker.png 'Severity: BLOCKER') Remove this hard-coded password. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2068)